cellprofiler/modules/relateobjects.py

```diff
@@ -828,17 +828,40 @@ def get_child_measurement_columns(self, pipeline):
 
         return columns
 
-    def get_measurement_columns(self, pipeline):
-        """Return the column definitions for this module's measurements"""
+    def get_saved_child_measurement_columns(self, pipeline):
+        """Return measurements for saved child objects"""
         if self.wants_child_objects_saved:
             columns = super(RelateObjects, self).get_measurement_columns(
-                pipeline,
-                additional_objects=[
-                    (self.y_name.value, self.output_child_objects_name.value)
-                ],
-            )
+                pipeline,additional_objects=[(self.y_name.value, self.output_child_objects_name.value)])
+            columns_to_return = []
+            for eachcolumn in columns:
+                if eachcolumn[0] == self.output_child_objects_name.value:
```

Comment:
Should this `if..elseif` be two conditions in an `or` statement, since both result in the same operation?